packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.vue

```diff
@@ -201,19 +221,48 @@ const { requiredMarkerClass, requiredTypeClass } = useRequired(props);
     }
   }
 
-  &:has(.onyx-select__input:enabled:focus) {
-    .onyx-select {
-      &__wrapper {
-        --border-color: var(--onyx-color-base-primary-500);
-        outline: var(--onyx-spacing-4xs) solid var(--onyx-color-base-primary-200);
+  &__loading {
+    color: var(--onyx-color-text-icons-primary-intense);
+  }
+
+  &__footer {
+    width: 100%;
+    color: var(--onyx-color-text-icons-neutral-soft);
+  }
+
+  &--editable {
+    .onyx-select__wrapper:has(.onyx-select__input:enabled) {
+      cursor: pointer;
+      // default hover
+      &:hover {
+        --border-color: var(--onyx-color-base-primary-400);
+        .onyx-select__icon {
+          color: var(--onyx-color-text-icons-primary-medium);
+        }
       }
+    }
+    // default focus
+    &:has(.onyx-select__input:enabled:focus) {
+      .onyx-select {
+        &__wrapper {
+          --border-color: var(--onyx-color-base-primary-500);
+          outline: var(--onyx-spacing-4xs) solid var(--onyx-color-base-primary-200);
+        }
 
-      &__icon {
-        color: var(--onyx-color-text-icons-primary-intense);
+        &__icon {
+          color: var(--onyx-color-text-icons-primary-intense);
+        }
       }
     }
   }
-  &:has(&__input:disabled) {
+
+  // readonly focus
+  &--readonly:has(.onyx-select__input:enabled:focus) .onyx-select__wrapper {
```

Comment:
Would prefer to use the `:enabled:read-only` CSS pseudo selector.
(:enabled is needed because :read-only will also apply to disabled)

Reply:
see other comment